aaad/validators.py

```diff
@@ -1,5 +1,73 @@
+#!/usr/bin/python
+
+from __future__ import print_function
+import utils
+import mesos
+import os
+import sys
+import json
+import yaml
+import re
+
+quota_file = os.getenv('QUOTA_FILE', '')
```

Comment:
what happens if quota_file is `''`? Perhaps it should return true at the start of validate.